setup.py

```diff
@@ -39,12 +39,18 @@
 VSC_INSTALL_REQ_VERSION = '0.9.19'
 
 PACKAGE = {
-    'version': '2.4.18',
+    'version': '2.5.0',
```

Comment:
Let's go with 2.5.0, no need to have a deep philosophical discussion about it. :)